portal/src/RoleAndGroupsLayout.module.css

```diff
@@ -24,7 +24,11 @@
 }
 
 .header {
-  @apply self-stretch;
+  @apply flex flex-row justify-between;
+}
+
+.breadcrumb {
+  @apply flex-1-0-auto;
```

Comment:
I think what you want is:
- Take all remaining spaces (grow = 1)
- But not more than remaining spaces (shrink = 1)

Therefore what you want should be flex-1.